vector/src/main/res/layout/fragment_space_list.xml

```diff
@@ -9,6 +9,7 @@
         android:id="@+id/groupListView"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
+        android:minHeight="195dp"
```

Comment:
when animation are disabled (for ui tests) for some reason dialog is not properly measure in 50% chances. This min height is the sum of height of 3 items in list and this is minimal amount of items to be shown for non-empty staye (all chat, 1 space, create space)

Reply:
Strange. Thanks for explaining.